tools/SetupFlow/DevHome.SetupFlow/Strings/en-us/Resources.resw

```diff
@@ -1979,4 +1979,28 @@
     <value>Select an extension provider before continuing.</value>
     <comment>Guidance text to instruct user to select a Quickstart provider before continuing</comment>
   </data>
+  <data name="QuickstartPlaygroundReferenceSample.Content" xml:space="preserve">

```

Comment:
Are these first 2 resources used?

Reply:
Good catch, those were left behind from an attempt at showing this as a button. Deleted.